tests/common/plugins/loganalyzer/loganalyzer.py

```diff
@@ -57,8 +58,9 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         """
         # Enable logrotate cron task back
         self.ansible_host.command("sed -i 's/^#//g' /etc/cron.d/logrotate")
-        # Enable logrotate systemd timer
-        self.ansible_host.command("systemctl start logrotate.timer")
+        # Enable logrotate systemd timer by best effort. The reason is that logrotate.timer service is not
+        # available in older version like 201911.
+        self.ansible_host.command("systemctl start logrotate.timer", module_ignore_errors=True)
```

Comment:
Same as above comment.

Reply:
It is not convenient to get the image version in this plugin. Need some dirty changes to have that information available. It would be cleaner to use a best effort method here.